contracts/SBCDepositContract.sol

```diff
@@ -264,12 +265,11 @@ contract SBCDepositContract is
      * NOTE: This function signature is hardcoded in the Gnosis execution layer clients. Changing this signature without updating the
      * clients will cause block verification of any post-shangai block to fail. The function signature cannonical spec is here
      * https://github.com/gnosischain/specs/blob/master/execution/withdrawals.md
-     * @param _deprecatedUnused Previously `maxFailedWithdrawalsToProcess` currently deprecated and ignored
```

Comment:
Does solhint require to remove this line? If no, please keep. If yes, then convert to non param line we need the context of that that first variable refers to